src/ext/java/org/opentripplanner/ext/siri/AffectsMapper.java

```diff
@@ -0,0 +1,430 @@
+package org.opentripplanner.ext.siri;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.opentripplanner.framework.time.ServiceDateUtils;
+import org.opentripplanner.routing.alertpatch.EntitySelector;
+import org.opentripplanner.routing.alertpatch.StopCondition;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+import org.opentripplanner.transit.model.timetable.Trip;
+import org.opentripplanner.transit.model.timetable.TripOnServiceDate;
+import org.opentripplanner.transit.service.TransitService;
+import uk.org.ifopt.siri20.StopPlaceRef;
+import uk.org.siri.siri20.AffectedLineStructure;
+import uk.org.siri.siri20.AffectedOperatorStructure;
+import uk.org.siri.siri20.AffectedRouteStructure;
+import uk.org.siri.siri20.AffectedStopPlaceStructure;
+import uk.org.siri.siri20.AffectedStopPointStructure;
+import uk.org.siri.siri20.AffectedVehicleJourneyStructure;
+import uk.org.siri.siri20.AffectsScopeStructure;
+import uk.org.siri.siri20.DataFrameRefStructure;
+import uk.org.siri.siri20.FramedVehicleJourneyRefStructure;
+import uk.org.siri.siri20.LineRef;
+import uk.org.siri.siri20.NetworkRefStructure;
+import uk.org.siri.siri20.OperatorRefStructure;
+import uk.org.siri.siri20.RoutePointTypeEnumeration;
+import uk.org.siri.siri20.StopPointRef;
+import uk.org.siri.siri20.VehicleJourneyRef;
+
+/**
+ * Maps a {@link AffectsScopeStructure} to a list of {@link EntitySelector}s
+ */
+public class AffectsMapper {
```

Comment:
I guess this is not new code? You just moved it from other class?

Reply:
Yes, they are extracted from [`SiriAlertsUpdateHandler`](https://github.com/opentripplanner/OpenTripPlanner/pull/4755/files#diff-fc53da460f84be7517cb2a81b0d1efc810565ad384fd76f2f4c7ed2a4bd919af)